src/components/BlockSwitcher/BlockSwitcher.tsx

```diff
@@ -13,14 +13,25 @@ export const BlockSwitcher = ({ children }: BlockSwitcherProps) => {
   if (!children.length || children.length <= 1) {
     throw new Error(BlockSwitcherErrorMessage);
   }
+
+  /**
+   * convert names with spaces to valid aria-controls values
+   */
+  const convertNameToValue = (name: string) => {
```

Comment:
Let's use a unique ID here to also fix the duplicate ID issue that can occur on some pages.

```
import { Children, useId } from 'react';

// ...

/**
 * Create unique id so that pages with BlockSwitchers that
 * use the same names don't create duplicate tab/panel ids
 */
const uniqueId = useId();

/**
 * convert names with spaces to valid aria-controls values
 */
const convertNameToValue = (name: string) => {
  return `${name.split(' ').join('-').toLowerCase()}-${uniqueId}`;
};
```